### ISSUE TITLE
Cannot reproduce GUI segmentation results in code for custom model.

### ISSUE THREAD

**Issue by kondratev-alexn** [MouseLand/cellpose#649]
I am using cellpose to segment cell nuclei in my data. Available models didn't produce satisfactory results so I labelled some cells, trained custom model and got good segmentation with it in GUI. However, when I tried running my model in code, I got worse segmentation results. I tried looking into the source, tried different diameter sizes, but could not resolve the issue. The previously opened issues with similar problems didn't help me.

The code I used for calculating masks:

```
model_path = "C:/Users/ALEX/.cellpose/models/phase_nucleus_segmentation"
master_diam = 53.7

channels = [[0,0]]
model = models.CellposeModel(gpu=True, pretrained_model=model_path)
imgs = [im for im in seq["image"][:5]] # seq["image"] is an iterator that yields1024*1024 images
masks, flows, styles = model.eval(imgs, diameter=master_diam, channels=channels)

diams = master_diam
dirsave = "save_dir"
files = [os.path.join(dirsave, f"{i}") for i in range(len(masks))]
io.masks_flows_to_seg(imgs, masks, flows, diams, files, channels)
io.save_to_png(imgs, masks, flows, files)
```

The worrying thing is, if I load produced _seg.npy file into GUI, it displays bad segmentation. But if I try to run the same model again on the image loaded from _seg.npy, the result doesn't change. However, when I load the same image again from the original image file, the model produces good segmentation again. I assume its connected to normalization or image size.

Here is segmentation result produced in the GUI:
![image](https://user-images.githubusercontent.com/15947990/215081902-be090e0b-bdb6-492c-b117-83fe154e5c02.png)

And here is one from _seg.npy file for the same image:
![image](https://user-images.githubusercontent.com/15947990/215082666-97e5ab22-f315-4179-9343-407188ea9928.png)

Here is the log file after running code with logger:
[run.log](https://github.com/MouseLand/cellpose/files/10518638/run.log)

And here is the same log file after loading _seg.npy segmentation, selecting my custom model and running it.
[run.log](https://github.com/MouseLand/cellpose/files/10518659/run.log)

How can I reproduce results from GUI in my code?

**Comment by carsen-stringer** [comment]
the bottom image looks brighter than the top image, are you sure you're loading in the same image? Cellpose will not modify the image in the notebook in the function

here is how the GUI calls the eval function FYI: https://github.com/MouseLand/cellpose/blob/03e02bca6abf15c4532088cec62a927923a8bf03/cellpose/gui/gui.py#L1770

**Comment by carsen-stringer** [comment]
not sure if this issue is related? unfortunately I can't reproduce that issue on my own computer

https://github.com/MouseLand/cellpose/issues/639

**Comment by kondratev-alexn** [comment]
I found out that there was implicit normalization in my part of code. After turning it off the results matched the one in the GUI. Thanks for the help, I was looking at the wrong direction.
